core/services/vrf/v2/listener_v2_log_listener.go

```diff
@@ -414,41 +419,28 @@ func (lsn *listenerV2) handleRequested(requested []RandomWordsRequested, request
 func numReplayBlocks(requestTimeout time.Duration, chainID *big.Int) int64 {
 	var timeoutSeconds = int64(requestTimeout.Seconds())
 	switch chainID.String() {
-	case "1": // eth mainnet
-	case "3": // eth ropsten
-	case "4": // eth rinkeby
-	case "5": // eth goerli
-	case "11155111": // eth sepolia
+	case "1", "3", "4", "5", "11155111": // eth mainnet, robsten, rinkeby, goerli, sepolia
```

Comment:
I kind of like the previous style of 1 chainId per line. It makes things much easier to read.

Reply:
I learned previous style is wrong. all of these went to `default:`

```
case "1": // eth mainnet
	case "3": // eth ropsten
	case "4": // eth rinkeby
	case "5": // eth goerli
```

Reply:
Ah. This style only works for languages with explicit `break`. Is it possible to something like below?
```go
case
  "1", // eth mainnet
  "3", // eth ropsten
  "4", // eth rinkeby
  "5": // eth goerli
```

Reply:
sure can change it